Gemfile

```diff
@@ -5,6 +5,7 @@ source "https://rubygems.org"
 git_source(:github) {|repo_name| "https://github.com/#{repo_name}" }
 
 gem "bundler"
-gem "jekyll", "3.9.0"
+gem "webrick"
+gem "jekyll"
```

Comment:
Not sure if it's necessary to lock down, since we checkin the .lock file once it's built and is running?